### ISSUE TITLE
Flow errors in project files

### ISSUE THREAD

**Issue by schumannd** [react-native-community/react-native-device-info#896]
I get:

```
Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ node_modules/react-native-device-info/src/index.js.flow:18:32

Unexpected token {, expected the token =

     15│   [key: string]: boolean,
     16│ };
     17│
     18│ export type AsyncHookResult<T> {
     19│   loading: boolean,
     20│   result: T | -1 | false | 'unknown',
     21│ }
```

and some other errors.

Using
"react-native-device-info": "5.3.0",
"react-native": "0.61.4",
"flow-bin": "0.105.2",

**Comment by matteodanelli** [comment]
It's already merged on [master](https://github.com/react-native-community/react-native-device-info/pull/886), if you need.

Else, just wait for next release, which will include the fix.

**Comment by mikehardy** [comment]
Sorry for the delay you two!
Released as 5.3.1 https://github.com/react-native-community/react-native-device-info/releases/tag/v5.3.1
